src/Hosted/.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
```

Comment:
@erolaliyev Can you push a commit which changes the .gitignore file by adding the `src/Hosted/.vs` path to it and remove the files that were added to the repo from that folder?